File: 04_Permissions/NormalPermissions/app/src/main/java/com/codingblocks/normalpermissions/MainActivity.java
package com.codingblocks.normalpermissions;

import android.annotation.SuppressLint;
import android.content.Context;
import android.net.ConnectivityManager;
import android.net.NetworkInfo;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    Button btnStatusBar;
    TextView tvStatus;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnStatusBar = findViewById(R.id.btnStatusBar);
        tvStatus = findViewById(R.id.tvStatus);

        btnStatusBar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                ConnectivityManager cm =
                        (ConnectivityManager) getSystemService(CONNECTIVITY_SERVICE);
                NetworkInfo netInfo = cm.getActiveNetworkInfo();
                boolean isConnected = netInfo != null &&
                        netInfo.isConnected();

                tvStatus.setText(isConnected ? "CONNECTED" : "DISCONNECTED");

            }
        });
    }
}

File: 12_Fragments/DynamicFragments/app/src/main/java/com/codingblocks/dynamicfragments/MainActivity.kt
package com.codingblocks.dynamicfragments

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)



        btnFruits.setOnClickListener {
            supportFragmentManager
                    .beginTransaction()
                    .replace(R.id.flFragContainer, FruitsFragment())
                    .commit()
        }

        btnVegetables.setOnClickListener {
            supportFragmentManager
                    .beginTransaction()
                    .replace(R.id.flFragContainer, VegetablesFragment())
                    .commit()
        }
    }
}
